### PR TITLE
Fix/3559 better handle autocomplete position

### PR DESCRIPTION
**Pull request by gbonaspetti**
#3559 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
~- [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)~ tests are too complexe and specifique to be done 
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/src/util/tinymceInit.js [hunk 1]
```diff
@@ -130,15 +130,15 @@ import { htmlCodeToDocumentFragment } from 'tracim_frontend_lib'
...
+
+          const topPosition = (isFullscreen ? $editor.getContainer().offsetTop : 0) + nodePosition.top + toolbarPosition.height()
+
           return {
-            top: (isFullscreen ? $editor.getContainer().offsetTop + nodePosition.top : nodePosition.top) + toolbarPosition.height(),
-            isSelectionToTheTop: nodePosition.top === 0,
-            selectionHeight: (nodeOffset.top * 2) + nodeHeight,
+            top: topPosition,
+            bottom: (isFullscreen ? $editor.getContainer().offsetTop : 0) + nodePosition.bottom + toolbarPosition.height(),
+            isSelectionToTheTop: topPosition < 280, // INFO - GB - 2021-07-06 - 280px is the autocomplete height
```
Comment (raphj) [inline frontend/src/util/tinymceInit.js:141]:
```suggestion
            isSelectionToTheTop: topPosition < AUTOCOMPLETE_HEIGHT,
```

I suggest replacing the comment by a constant with a descriptive name :-)

Do you know where it is computed? Maybe there is a way to share this value in the two places?

**Review by raphj — APPROVED** [review]

File: frontend/src/util/tinymceInit.js (hunk 1 above)
Comment (gbonaspetti) [reply frontend/src/util/tinymceInit.js:141]: The real value is "auto", it depends on the number of elements. I tested with the number max and I got the value. Not ideal, but better than the last one that didn't works. There is an issue to better handle this autocompletion: https://github.com/tracim/tracim/issues/3544 I'm gonna add a point about it.

Comment (gbonaspetti) [reply frontend/src/util/tinymceInit.js:141]: I'm gonna add the constant too

Comment (raphj) [reply frontend/src/util/tinymceInit.js:141]: good :+1:

**Comment by raphj** [comment]
Tested on branch.

**Pull request merged**